src/MapViewController.swift

```diff
@@ -17,7 +17,21 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     var lastSetPoint: TGGeoPoint?
     var shouldShowCurrentLocation = false
     public var shouldFollowCurrentLocation = false
-    public var findMeButton = MapViewController.createFindMeButton()
+    public var findMeButton = UIButton(type: .Custom)
```

Comment:
Just to clarify - answer is yes cuz Swift. There may be a cleaner way of doing it though. I'll look into it for future cleanup.